src/frontend/Dockerfile

```diff
@@ -1,5 +1,12 @@
 FROM node:12-slim
 
+# Update debian stretch repositories
+# Fixes upstream dependency of the node:12-slim base image
```

Comment:
Is there a card to upgrade this container, or will that be done as part of the front-end refresh? We should use something modern for the new work.

Reply:
As part of the front-end refresh but it wasn't documented, just added issue https://github.com/azavea/echo-locator/issues/606 to ensure that work is captured!